http-gateway/web/src/containers/things/things-i18n.js

```diff
@@ -98,6 +103,11 @@ export const messages = defineMessages({
     defaultMessage:
       'The resource was updated successfully, changes will be applied once the device is online.',
   },
+  resourceWasDeletedOffline: {
+    id: 'things.resourceWasUpdatedOffline',
+    defaultMessage:
+      'Deleting of the resource was scheduled, but it will be executed once the device is online.',

```

Comment:
`Deleting of the resource was scheduled, it will be deleted once the device is online.`